test_utils/src/cluster.rs

```diff
@@ -643,6 +643,24 @@ impl AuthorityDetails {
         ProposerClient::new(channel)
     }
 
+    /// Creates a new configuration client that connects to the corresponding client.
+    /// This should be available only if the internal consensus is disabled. If
+    /// the internal consensus is enabled then a panic will be thrown instead.
```

Comment:
I don't think I see any risk in letting this be accessible with the internal consensus enabled. Do you?

Reply:
It won't be able to connect as the grpc server only starts when internal consensus is disabled [here](https://github.com/MystenLabs/narwhal/blob/main/primary/src/primary.rs#L331-L346).